is-outside-dir-win32.js

```diff
@@ -2,9 +2,14 @@
 
 const path = require('path');
 const minimatch = require('minimatch');
-
-const dot = { dot: true };
-
-module.exports = function(dir, filename) {
-    return !minimatch(path.resolve(dir, filename), path.join(dir, '**'), dot);
+const minimatchOptions = {
+    dot: true,
+    nocase: true // win32 should be case insensitive matches
 };
+
+module.exports = {
+    isOutsideDir(dir, filename) {
+        return !minimatch(path.resolve(dir, filename), path.join(dir, '**'), minimatchOptions)
```

Comment:
Fwiw, this still seems to the wrong result for me on Windows when `dir` is relative ... I think if we want to resolve one path, we should also do so for the other?

```suggestion
        return !minimatch(path.resolve(dir, filename), path.resolve(dir, '**'), minimatchOptions)
```